sei_dbt/README.md

```diff
@@ -79,17 +108,155 @@ Then, it will:
+
+
+To **infer variables from `profiles.yml`** in your **dbt model file**, you need to use **Jinja** templating and the `target` object, which exposes information from `profiles.yml`.
+
+
+#### Step 2: Access the Variables in the Model File
+
+
+Here’s an example of how you might use the `data_path` variable inside a dbt model:
+
+```sql
+-- models/staging/my_staging_model.sql
+
+WITH raw_data AS (
+
+    -- Use the data_path from profiles.yml
+    SELECT *
+    FROM read_csv_auto('{{ target.external_root }}/my_file.csv')
+
+),
+
+final_data AS (
+
+    SELECT
+        *
+    FROM raw_data
+
+)
+
+SELECT * FROM final_data;
+```
+
+- **`{{ target.vars.data_path }}`**: This Jinja expression accesses the `data_path` variable from `profiles.yml`. When you run dbt with the `--target dev` flag, it will dynamically insert `"data/dev"`, and for `--target prod`, it will insert `"data/prod"`.
+
+In this case, the model will:
+1. **Use the `data_path`** defined in `profiles.yml` to load the appropriate CSV file based on the environment (e.g., `data/dev/my_file.csv` in dev, `data/prod/my_file.csv` in prod).
+
+
+#### Source:
+- [Profiles.yml Reference](https://docs.getdbt.com/docs/configure-your-profile)
+   
+   
+4. **Configure project:**   Project-Specific Configuration
+
+The `dbt_project.yml` file is the configuration file specific to a **single dbt project**. It contains settings related to that particular project, such as the project's name, paths to models, seeds, and specific **variables** (used in transformations and models).
+
+This file is usually placed in the root of the dbt project directory and is included in version control because it’s tied to the project itself.
+
+#### Example of `dbt_project.yml`:
+```yaml
+name: 'my_dbt_project'
+version: '1.0.0'
+profile: 'my_profile'
+
+# Define the structure and directories for models
+model-paths: ["models"]
+seed-paths: ["seeds"]
+
+# Project-specific variables
+vars:
+  some_variable: 'value'
+  environment: 'dev'  # Example environment variable
+```
+
+#### Key Usage:
+- **Variables**: You can define project-specific variables under `vars`. These are useful when you need to reference configuration values in multiple places in your models, macros, or tests.
+  - These variables are **project-specific** and are **not meant to store sensitive information** like database credentials.
+- **Project Paths**: You can configure paths for models, seeds, and snapshots.
+- **Project-specific Settings**: This file defines what dbt will do within the scope of your project. The configurations are highly tailored to that particular project, making it version-controlled and tied to the specific project context.
+
+#### Source:
+- [dbt_project.yml Reference](https://docs.getdbt.com/reference/project-configs/dbt_project.yml)
+   
+   
+### **Comparison of `dbt_project.yml` vs. `profiles.yml`**
```

Comment:
This table and the best practices section that follows seems overly complicated for the project. Try to focus the README on project specific information. If we want to store our learnings about best practices for how to use dbt, that should probably be stored with the rest of the project files for now.

Reply:
Agree. Delete it.